src/extension.ts

```diff
@@ -164,3 +181,27 @@ function convertLogLevel(commonLogLevel: string) {
             return "FINE";
     }
 }
+
+function collectUserSettings(): string[] {
+    const settings: string[] =  [];
+    if (vscode.workspace.getConfiguration().get("java.debug.settings.showHex")) {
+        settings.push("--show_hex");
```

Comment:
Is the `--` necessary in here. This is not console parameter. In this case, we should just naming setting name for setting values.